src/shared/api/index.ts

```diff
@@ -32,3 +32,18 @@ export function useModels(
 
   return { models, ...rest }
 }
+
+export function useModelGroups(
+  id: Ref<number>,
+  idMarks: Ref<number>,
+  useFetchOptions: UseFetchOptions = {}
+) {
+  const { data, ...rest } = useAutoRia<CategoriesResponse>(
```

Comment:
Hmm, I understand that the response scheme is the same as for categories, but please create ModelGroupsResponse.